webpack/assets/javascripts/react_app/components/FactCharts/__test__/selectors.test.js

```diff
@@ -1,12 +1,12 @@
 import { STATUS } from '../../../constants';
-import { chartData, modalToDisplay, id, key } from '../FactChart.fixtures';
+import { chartData, modalToDisplay, id, key } from '../fixtures';
 import {
   selectHostCount,
   selectFactChart,
   selectDisplayModal,
   selectFactChartData,
   selectFactChartStatus,
-} from '../FactChartSelectors';
+} from '../selectors';
```

Comment:
I believe the selectors' test can be removed as Redux code [can be treated as an implementation detail of the app](https://redux.js.org/usage/writing-tests#guiding-principles)